src/var/mash.jl

```diff
@@ -0,0 +1,57 @@
+# MinHash
+# ========
+#
+# Functions to get MASH distance for MinHash sketches
+#
+# see DOI: 10.1186/s13059-016-0997-x
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+function jaccarddistance(sketch1::MinHashSketch, sketch2::MinHashSketch)
```

Comment:
@kescobo I expected the Distances methods would be slower since your function is tailored to the type.